### ISSUE TITLE
Upgrade PS from 8.0.4 to 8.1.0 - Error upgrading Doctrine schema

### ISSUE THREAD

**Issue by PeNov** [PrestaShop/PrestaShop#33021]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hello,

during the upgrade PS 8.0.4 to PS 8.1.0 these errors are show's:

Error during database upgrade. You may need to restore your database.
Error upgrading Doctrine schema

Log:
[log_804_to_810.txt](https://github.com/PrestaShop/PrestaShop/files/11880937/log_804_to_810.txt)

Seams that PS works as version 8.1.0 now. It is not fully tested.

Are you able suggest me necessary steps?

For example:
A) Roll back to 8.0.4 and wait to new version
B) Repair something in the database (run commands) for correct working of e-shop.

Thank you
Best regards
Petr

### Expected behavior

Upgrade without errors.

### Steps to reproduce

All modules was upgraded to newest including 1-Click Upgrade module (4.16.0)
Go to 1-Click Upgrade and Upgrade - standard as many times before.

### PrestaShop version(s) where the bug happened

8.0.4 to 8.1.0

### PHP version(s) where the bug happened

7.4.33

### If your bug is related to a module, specify its name and its version

autoupgrade v4.16.0

**Comment by Hlavtox** [comment]
https://github.com/laravel/framework/issues/24711#issuecomment-401566425

```
This happens because utf8mb4 uses 4 bytes per character, and the email column has a 255 character length which is greater than the limit 767 bytes. 255 x 4bytes = 1020b.

To fix this the email column length should be at most 191. 191 x 4 bytes = 764b.
```

Solution = Use utf8 instead utf8mb4 OR change `name` column length to 191 chars.

Ping @PrestaShop/committers

**Comment by sallemiines** [comment]
Hello @PeNov 
Could you please try the proposed solution  `Use utf8 instead utf8mb4 OR change name column length to 191 chars.` as @[Halvatox](https://github.com/Hlavtox) said .

Waiting for your feedback ! 
Thank you !

**Comment by darkf3d3** [comment]
I got the same error when upgrading from 1.7.7.7 to 8.1 (the same website it's upgraded correctly to 8.0.4).
I tried performing the upgrade 3 times, and I get always the same errors:
`All files upgraded. Now upgrading database...

Updating database schema...
Executing: CREATE TABLE ps_authorized_application (id_authorized_application INT UNSIGNED AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, description LONGTEXT NOT NULL, UNIQUE INDEX UNIQ_475B9BA55E237E06 (name), PRIMARY KEY(id_authorized_application)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB
15:09:52 CRITICAL [console] Error thrown while running command "prestashop:schema:update-without-foreign --env=prod". Message: "An exception occurred while executing 'CREATE TABLE ps_authorized_application (id_authorized_application INT UNSIGNED AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, description LONGTEXT NOT NULL, UNIQUE INDEX UNIQ_475B9BA55E237E06 (name), PRIMARY KEY(id_authorized_application)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB':
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes" ["exception" => Doctrine\DBAL\Exception\DriverException { …},"command" => "prestashop:schema:update-without-foreign --env=prod","message" => """ An exception occurred while executing 'CREATE TABLE ps_authorized_application (id_authorized_application INT UNSIGNED AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, description LONGTEXT NOT NULL, UNIQUE INDEX UNIQ_475B9BA55E237E06 (name), PRIMARY KEY(id_authorized_application)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB':\n \n SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes """]
In AbstractMySQLDriver.php line 128:
An exception occurred while executing 'CREATE TABLE ps_authorized_applicati
on (id_authorized_application INT UNSIGNED AUTO_INCREMENT NOT NULL, name VA
RCHAR(255) NOT NULL, description LONGTEXT NOT NULL, UNIQUE INDEX UNIQ_475B9
BA55E237E06 (name), PRIMARY KEY(id_authorized_application)) DEFAULT CHARACT
ER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB':
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was t
oo long; max key length is 767 bytes
In Exception.php line 18:
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was t
oo long; max key length is 767 bytes
In PDOConnection.php line 141:
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was t
oo long; max key length is 767 bytes
prestashop:schema:update-without-foreign [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--id_shop [ID_SHOP]] [--id_shop_group [ID_SHOP_GROUP]] [--]
[DEV] ajax request : rollback

Errors
Error during database upgrade. You may need to restore your database.
Error upgrading Doctrine schema`

**Comment by PeNov** [comment]
Hello @sallemiines and @Hlavtox,

Yes sure, I can make test.

But I need more information’s about these two scenarios.

Should I go back to version PS 8.0.4 firstly without any changes?

**A)	Change config/database.php**
The folder config does not contain file database.php

**B)	change name column length to 191 chars**
I can try it. But table ps_authorized_application does not exit. Where I makes this change? On the 1-click update script?

Thank you
Best regards
Petr

**Comment by mflasquin** [comment]
Hi @PeNov you can try to change in src/PrestaShopBundle/Entity/AuthorizedApplication.php::56

**Comment by kpodemski** [comment]
@darfk3d3 @PeNov 

could you guys provide us with more information about your DB server? You can ask hosting provider, or go to the phpMyAdmin or some other database GUI, so that we know what is the default encoding on your server, not the encoding you have in the tables.

This is what I see after login:
https://capture.dropbox.com/DkAww4gg5qhLuLwH

After getting into the database, you can also see whether you have MyISAM or InnoDb tables.

**Comment by darkf3d3** [comment]
@kpodemski 
Sure, here's mine:
![Screenshot 2023-06-28 094738](https://github.com/PrestaShop/PrestaShop/assets/42684711/02316906-e624-4050-9e3b-7acfa277e899)

**Comment by mflasquin** [comment]
Hi @darfk3d3, can you provide some other informations from tables like this :

![image](https://github.com/PrestaShop/PrestaShop/assets/6594752/647c1630-f104-42a0-a72b-6237711fe19f)

**Comment by darkf3d3** [comment]
@mflasquin 
![image](https://github.com/PrestaShop/PrestaShop/assets/42684711/25fd6dd6-43ee-41ba-848f-4c37e9be2c08)
![image](https://github.com/PrestaShop/PrestaShop/assets/42684711/0e0933d2-b81f-4bc8-90e7-f5b16f646539)
This is just a part, let me know if you need more tables...

**Comment by kpodemski** [comment]
@darfk3d3 all good, thanks

**Comment by PeNov** [comment]
Hello Krystian @kpodemski ,
Here is screen after login to phpMyAdmin:
![image](https://github.com/PrestaShop/PrestaShop/assets/38213520/aa329eef-6068-4ccc-a0d7-da5eb2423c8b)

Tables have more type of the collation. For example: utf8mb4_general_ci, utf8_unicode_ci, utf8_general_ci, latin1_swedish_ci

**MySQL info:**
Verze MySQL: 10.1.48-MariaDB-0+deb9u2
Engine MySQL: InnoDB
Driver MySQL: DbPDO

**Server info:**
Apache/2.4.56 (Debian)
PHP version: 7.4.33
Memory limit: 256M
Max. time execution: 180
Max. size of the uploaded file: 60M

Thank you
Best regards
Petr

**Comment by ChillCode** [comment]
Without testing I guess this new table requires a really big INDEX and is reaching the limits of your MariaDB version:

In the link you'll find an explanation:

https://dev.mysql.com/doc/refman/8.0/en/innodb-limits.html

To check row format:

```
SHOW VARIABLES LIKE 'innodb_default_row_format';

Result:

innodb_default_row_format | dynamic
```


If you can upgrade MariaDB I think problem will be solved easily since the default ROW FORMAT is DYNAMIC.

https://mariadb.com/kb/en/innodb-dynamic-row-format/

As stated in the previous link may this works for you both if you cannot upgrade:

```

## Innodb settings to bypass error of max size 737
innodb-file-format=barracuda
innodb-file-per-table=ON
innodb-large-prefix=ON
## Above 3 didnot work so i added below
innodb_default_row_format = 'DYNAMIC'

```

[Source](https://stackoverflow.com/a/57465235)

May the docs should reflect this new requirement to upgrade to a greater MariaDB version that allows 3072bytes as stated in the links posted.

**Comment by joecoolio66** [comment]
We don't understand... is this a bug?  Is this a new requirement?  Why is there issue here?

**Comment by darkf3d3** [comment]
> If you can upgrade MariaDB I think problem will be solved easily since the default ROW FORMAT is DYNAMIC.

I asked my hosting about upgrading MariaDB version. They said me that it can be done, but it will be done serverwide and the operation is not reversible, so I'm not sure about this because I have dozen of website of the server, and I'm not sure about MariaDB versions compatibility.

**Comment by ghost** [comment]
Im also getting "database" errors at the end of auto-upgrade, doesn't really seem to be database related though

PHP Version 8.1.16
MySQL version: 10.4.30-MariaDB

`
[INTERNAL] /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 180 - Symfony\Component\Config\Exception\LoaderLoadException: There is no extension able to load the configuration for "api_platform" (in "/home/devel/public_html/app/config/config.yml"). Looked for namespace "api_platform", found ""framework", "security", "twig", "monolog", "swiftmailer", "doctrine", "sensio_framework_extra", "prestashop", "translation_tools", "tactician", "fos_js_routing"" in /home/devel/public_html/app/config/config.yml (which is being imported from "/home/devel/public_html/app/config/config_prod.yml"). #0 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php(104): Symfony\Component\Config\Loader\FileLoader->doImport() #1 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/FileLoader.php(66): Symfony\Component\Config\Loader\FileLoader->import() #2 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(197): Symfony\Component\DependencyInjection\Loader\FileLoader->import() #3 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(133): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->parseImports() #4 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/DelegatingLoader.php(40): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->load() #5 /home/devel/public_html/app/AppKernel.php(211): Symfony\Component\Config\Loader\DelegatingLoader->load() #6 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(697): AppKernel->registerContainerConfiguration() #7 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(593): Symfony\Component\HttpKernel\Kernel->buildContainer() #8 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(136): Symfony\Component\HttpKernel\Kernel->initializeContainer() #9 /home/devel/public_html/app/AppKernel.php(93): Symfony\Component\HttpKernel\Kernel->boot() #10 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/SymfonyAdapter.php(76): AppKernel->boot() #11 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php(44): PrestaShop\Module\AutoUpgrade\UpgradeTools\SymfonyAdapter->initAppKernel() #12 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php(87): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader80->initConstants() #13 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader81.php(46): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader->doUpgrade() #14 /home/devel/public_html/modules/autoupgrade/classes/TaskRunner/Upgrade/UpgradeDb.php(43): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader81->doUpgrade() #15 /home/devel/public_html/admin3756utwoe9xinaeuvsl/autoupgrade/ajax-upgradetab.php(53): PrestaShop\Module\AutoUpgrade\TaskRunner\Upgrade\UpgradeDb->run() #16 {main}
`

**Comment by ChillCode** [comment]
> > If you can upgrade MariaDB I think problem will be solved easily since the default ROW FORMAT is DYNAMIC.
> 
> I asked my hosting about upgrading MariaDB version. They said me that it can be done, but it will be done serverwide and the operation is not reversible, so I'm not sure about this because I have dozen of website of the server, and I'm not sure about MariaDB versions compatibility.

My response is only what MySQL/MariaDB docs says about that error you're getting, both with 10.1.38, and I wouldn't do an upgrade on a production server neither.

On a staging site so you can try to upgrade again and then see the results and take a decision.

But PrestaShop may change column definition to solve the issue.

**Comment by joecoolio66** [comment]
Apparently we use percona... has there been any reported issues with that DB?

**Comment by ChillCode** [comment]
Percona is also a MySQL distribution but you can wait till @mflasquin release the solution and then upgrade.

**Comment by xga91** [comment]
> Im also getting "database" errors at the end of auto-upgrade, doesn't really seem to be database related though
> 
> PHP Version 8.1.16 MySQL version: 10.4.30-MariaDB
> 
> `[INTERNAL] /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 180 - Symfony\Component\Config\Exception\LoaderLoadException: There is no extension able to load the configuration for "api_platform" (in "/home/devel/public_html/app/config/config.yml"). Looked for namespace "api_platform", found ""framework", "security", "twig", "monolog", "swiftmailer", "doctrine", "sensio_framework_extra", "prestashop", "translation_tools", "tactician", "fos_js_routing"" in /home/devel/public_html/app/config/config.yml (which is being imported from "/home/devel/public_html/app/config/config_prod.yml"). #0 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php(104): Symfony\Component\Config\Loader\FileLoader->doImport() #1 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/FileLoader.php(66): Symfony\Component\Config\Loader\FileLoader->import() #2 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(197): Symfony\Component\DependencyInjection\Loader\FileLoader->import() #3 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(133): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->parseImports() #4 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/DelegatingLoader.php(40): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->load() #5 /home/devel/public_html/app/AppKernel.php(211): Symfony\Component\Config\Loader\DelegatingLoader->load() #6 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(697): AppKernel->registerContainerConfiguration() #7 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(593): Symfony\Component\HttpKernel\Kernel->buildContainer() #8 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(136): Symfony\Component\HttpKernel\Kernel->initializeContainer() #9 /home/devel/public_html/app/AppKernel.php(93): Symfony\Component\HttpKernel\Kernel->boot() #10 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/SymfonyAdapter.php(76): AppKernel->boot() #11 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php(44): PrestaShop\Module\AutoUpgrade\UpgradeTools\SymfonyAdapter->initAppKernel() #12 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php(87): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader80->initConstants() #13 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader81.php(46): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader->doUpgrade() #14 /home/devel/public_html/modules/autoupgrade/classes/TaskRunner/Upgrade/UpgradeDb.php(43): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader81->doUpgrade() #15 /home/devel/public_html/admin3756utwoe9xinaeuvsl/autoupgrade/ajax-upgradetab.php(53): PrestaShop\Module\AutoUpgrade\TaskRunner\Upgrade\UpgradeDb->run() #16 {main}`

I got same error message as you, did you manage to find anything ?

**Comment by AureRita** [comment]
Hi @PeNov 

Thank you for your report, I need to know some things with you : 

Did your issue seems like this one ? 

[Untitled_ Jul 3, 2023 12_08 PM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/85618f3a-9193-4601-8da7-dde9781cf70e)

Moreover, did your issue seems like this another issue ; https://github.com/PrestaShop/PrestaShop/issues/33041 

but at the end, I think this issue is too technical for me, ping @PrestaShop/prestashop-core-developers, can someone please reproduce this issue ? 

Thanks

**Comment by ghost** [comment]
> Im also getting "database" errors at the end of auto-upgrade, doesn't really seem to be database related though
> 
> PHP Version 8.1.16 MySQL version: 10.4.30-MariaDB
> 
> `[INTERNAL] /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 180 - Symfony\Component\Config\Exception\LoaderLoadException: There is no extension able to load the configuration for "api_platform" (in "/home/devel/public_html/app/config/config.yml"). Looked for namespace "api_platform", found ""framework", "security", "twig", "monolog", "swiftmailer", "doctrine", "sensio_framework_extra", "prestashop", "translation_tools", "tactician", "fos_js_routing"" in /home/devel/public_html/app/config/config.yml (which is being imported from "/home/devel/public_html/app/config/config_prod.yml"). #0 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php(104): Symfony\Component\Config\Loader\FileLoader->doImport() #1 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/FileLoader.php(66): Symfony\Component\Config\Loader\FileLoader->import() #2 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(197): Symfony\Component\DependencyInjection\Loader\FileLoader->import() #3 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php(133): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->parseImports() #4 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/DelegatingLoader.php(40): Symfony\Component\DependencyInjection\Loader\YamlFileLoader->load() #5 /home/devel/public_html/app/AppKernel.php(211): Symfony\Component\Config\Loader\DelegatingLoader->load() #6 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(697): AppKernel->registerContainerConfiguration() #7 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(593): Symfony\Component\HttpKernel\Kernel->buildContainer() #8 /home/devel/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(136): Symfony\Component\HttpKernel\Kernel->initializeContainer() #9 /home/devel/public_html/app/AppKernel.php(93): Symfony\Component\HttpKernel\Kernel->boot() #10 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/SymfonyAdapter.php(76): AppKernel->boot() #11 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php(44): PrestaShop\Module\AutoUpgrade\UpgradeTools\SymfonyAdapter->initAppKernel() #12 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php(87): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader80->initConstants() #13 /home/devel/public_html/modules/autoupgrade/classes/UpgradeTools/CoreUpgrader/CoreUpgrader81.php(46): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader->doUpgrade() #14 /home/devel/public_html/modules/autoupgrade/classes/TaskRunner/Upgrade/UpgradeDb.php(43): PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader\CoreUpgrader81->doUpgrade() #15 /home/devel/public_html/admin3756utwoe9xinaeuvsl/autoupgrade/ajax-upgradetab.php(53): PrestaShop\Module\AutoUpgrade\TaskRunner\Upgrade\UpgradeDb->run() #16 {main}`

Same problem with same errors
fresh 8.0.4 intall
8.0.4 to 8.1.0
Updating test with different php versions
Updating test with tow different updating modules
Failed...

**Comment by PeNov** [comment]
> 

Hello @AureRita, 

my error is described in the first paragraph of the bug report #33021. I have only Error upgrading Doctrine schema.

Your video showed different behavior. I have not Ajax error. I have only Doctrine schema error.
 
Bug report #33041 is different too from my report.

Best regards
Petr

**Comment by darkf3d3** [comment]
I update my situation, updating mariadb version from 10.1 to 10.11 solved the problem.
After doing that I was able to upgrade from PS1.7 to PS8.1 succesfully.

I just have a minor issue with back office left menu show names in English instead of Italian, but I don't think it could be something related to mariadb.

**Comment by kpodemski** [comment]
@darkf3d3 that's great! :)

regarding the back office menu, update your language pack, that should do it - go to International -> Translations and update Italian language :)

**Comment by darkf3d3** [comment]
@kpodemski Thanks, now it's 100% Italian. I didn't know that I need to do that after upgrade. It's an expecter behaviour that it's not done automatically during the upgrade?

**Comment by kpodemski** [comment]
@darkf3d3 I honestly don't know :) I used to do it manually, not only after the upgrade, etc., but regularly just to get fresh translations from time to time :)

**Comment by darkf3d3** [comment]
@kpodemski Good to know. I will do this too. Thanks ;)

**Comment by blvckcoder** [comment]
For those encountering this bug, from version 8.0.4 -> 8.1.0 and higher with doctrine database:

1. **Restore** your healthy backup
2. Simply **disable the psgdpr module**
3. Proceed with the **update using clickupgrade**
4. **Done** without errors

**Comment by maxadp** [comment]
I got the same problem, I can't update from 8.0.5 to 8.1.3, got everytime: "Error during database upgrade. You may need to restore your database." "Error upgrading Doctrine schema"

I try to use @mflasquin fix but in 8.0.5 I got no AuthorizedApplication.php file 
 in src/PrestaShopBundle/Entity/ so for me is not useful

I try also to disable psgdpr module, but got same error.

I can't update mysql or mariadb version.

What I can do?

**Comment by Pele4711** [comment]
Hi there,
same here. Just tried to upgrade from 8.1.1 to 8.1.4. Also the same Doctrine error. The psgdpr module uninstalled.
Any solution?
